ocaml/xapi/export_raw_vdi.ml

```diff
@@ -47,11 +47,16 @@ let localhost_handler rpc session_id vdi (req : Http.Request.t)
           let copy base_path path size =
             try
               debug "Copying VDI contents..." ;
-              Vhd_tool_wrapper.send ?relative_to:base_path
-                (Vhd_tool_wrapper.update_task_progress __context)
-                "none"
-                (Importexport.Format.to_string format)
-                s path size "" ;
+              if format = Qcow then
+                Qcow_tool_wrapper.send
+                  (Qcow_tool_wrapper.update_task_progress __context)
+                  s path size
+              else
+                Vhd_tool_wrapper.send ?relative_to:base_path
+                  (Vhd_tool_wrapper.update_task_progress __context)
+                  "none"
+                  (Importexport.Format.to_string format)
```

Comment:
It's better to use a `match` statement here (like the one below). In case more variants are added in the future, `if` would not raise an error/warning, but an explicit `match` would

Reply:
Yes if a new format is added it will fall through the else. I see the point of catching new format. I will do that way.